src/components/box/_box.scss

```diff
@@ -0,0 +1,14 @@
+$boxBorderColor: $graySecondary;
+
+.mint-box {
+  @include component;
+
+  border-radius: 22px;
```

Comment:
I am wondering why it's `22px`. Does it have something related to `$defaultLayoutPadding`?